docs/modules/ROOT/pages/maploader-and-mapstore.adoc

```diff
@@ -1,96 +1,171 @@
-= Building a MapStore for Hazelcast {hazelcast-cloud}
+= Build a MapStore for Hazelcast {hazelcast-cloud}
 :toclevels: 3
 :url-code-sample-mapstore: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/mapstore
-:description: pass:q[MapStore is an API for building a cache on top of Hazelcast. The MapStore interface includes methods that are triggered when operations are invoked on a map. You can implement your own logic in these methods to connect to an external data store, load data from it, and write data back to it. For example, you can use a MapStore to load data into your cluster from a MongoDB, MySQL, or PostgreSQL database.]
+:description: pass:q[The Hazelcast map module classes in the link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/map/package-summary.html[Hazelcast Root API] provide in-memory data stores. The MapStore module classes allow you to integrate data from external data sources, such as a database, file-based data store, or Kafka. This means that, if the requested value is not available in the in-memory data store, Hazelcast can attempt to load it from your external data sources. You can also use MapStore to store or delete values in your external data source. You can implement your own logic in the MapStore methods.]
```

Comment:
While in theory it's possible to implement a MapLoader/Store on top of file or Kafka, it's impractical and only really done on top of databases which support select, update, insert/upsert and delete statements (or their equivalent).

Reply:
removed mentions of file-based data store and Kafka from the example data sources